samples/java-eventsourced-shopping-cart/src/it/java/com/example/shoppingcart/domain/ShoppingCartIntegrationTest.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.example.shoppingcart.api;
```

Comment:
This is surprising. The test was defined in package `api`, but it was in directory `com/example/shoppingcart`. 

Noow moved to `com.example.shoppingcart.domain` which is the place where it gets generated.

Reply:
https://github.com/lightbend/akkaserverless-java-sdk/issues/678 I thought I fixed by moving in #675 though, strange.